lib/qb_integration/address.rb

```diff
@@ -15,5 +15,16 @@ def self.build(order_address)
     def self.hash_or_empty(hash, key)
       value = hash.nil? ? '' : hash[key]
     end
+
+    def self.as_flowlink_hash(qbo_address)
```

Comment:
hmm, :thinking: , I know this follows the integration design... but this seems to be an instance method for the address, it is a bit weird we set as a module function and pass the address as input, :thinking:  ... we should think about redesigning this